jac-uk/apply
# Feature/1209 Update character section of the application form

## What's included?
Closes #1209 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

[Example vacancy](https://jac-apply-develop--pr1229-feature-1209-update-lcscluww.web.app/vacancy/Biyjd07Xz2usL9yXjtjV/)

1. Apply for the vacancy and go to "Character information".
2. Check if the Character questions are updated and displayed correctly.
3. Check if the Welsh translation of the Character questions is updated.

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context

Demo:

https://github.com/user-attachments/assets/9d8fab90-c59b-4fb0-92c4-e590f7cea8bd

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
